JuiceMaker/JuiceMaker/Controller/JuiceMakerViewController.swift

```diff
@@ -6,8 +6,8 @@
 
 import UIKit
 
-class JuiceMakerViewController: UIViewController {
-    private let juiceMaker = JuiceMaker()
+class JuiceMakerViewController: UIViewController, ChangeStockProtocol {
```

Comment:
보통은 가독성의 문제로 extension으로 확장해서 구현해주기도 합니다!

```suggestion
class JuiceMakerViewController: UIViewController { }

extension JuiceMakerViewController: ChangeStockProtocol { }
```

Reply:
extension으로 확장해서 구현해주었습니다!